src/OrderByDescending.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
Comments from `src/OrderBy.js` apply to this file.

Reply:
Fixed